core/src/main/java/org/axonframework/eventsourcing/eventstore/AbstractEventStorageEngine.java

```diff
@@ -203,7 +204,7 @@ protected void handlePersistenceException(Exception exception, EventMessage<?> f
      * @param aggregateIdentifier The aggregate identifier to fetch a snapshot for
      * @return An optional with a serialized snapshot of the aggregate
```

Comment:
Don't forget to change the Javadoc. Also explain that the snapshot are returned in descending order of their sequence number.